code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -75,38 +77,58 @@
 	C.dna.remove_mutation(/datum/mutation/human/race)
 
 /datum/species/monkey/spec_unarmedattack(mob/living/carbon/human/user, atom/target, modifiers)
-	if(HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
-		if(!iscarbon(target))
-			return TRUE
-		var/mob/living/carbon/victim = target
-		if(user.is_muzzled())
-			return TRUE
-		var/obj/item/bodypart/affecting = null
-		if(ishuman(victim))
-			var/mob/living/carbon/human/human_victim = victim
-			affecting = human_victim.get_bodypart(human_victim.get_random_valid_zone(even_weights = TRUE))
-		var/armor = victim.run_armor_check(affecting, MELEE)
-		if(prob(25))
-			victim.visible_message(span_danger("[user]'s bite misses [victim]!"),
-				span_danger("You avoid [user]'s bite!"), span_hear("You hear jaws snapping shut!"), COMBAT_MESSAGE_RANGE, user)
-			to_chat(user, span_danger("Your bite misses [victim]!"))
-			return TRUE
-		var/obj/item/bodypart/head/mouth = user.get_bodypart(BODY_ZONE_HEAD)
-		victim.apply_damage(rand(mouth.unarmed_damage_low, mouth.unarmed_damage_high), BRUTE, affecting, armor)
-		victim.visible_message(span_danger("[name] bites [victim]!"),
-			span_userdanger("[name] bites you!"), span_hear("You hear a chomp!"), COMBAT_MESSAGE_RANGE, name)
-		to_chat(user, span_danger("You bite [victim]!"))
-		if(armor >= 2)
+	// If our hands are not blocked, dont try to bite them
+	if(!HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
+		// if we aren't an advanced tool user, we call attack_paw and cancel the preceeding attack chain
+		if(!ISADVANCEDTOOLUSER(user))
+			target.attack_paw(user, modifiers)
 			return TRUE
-		for(var/datum/disease/bite_infection as anything in user.diseases)
-			if(bite_infection.spread_flags & (DISEASE_SPREAD_SPECIAL | DISEASE_SPREAD_NON_CONTAGIOUS))
-				continue
-			victim.ForceContractDisease(bite_infection)
+		return ..()
+
+	// this shouldn't even be possible, but I'm sure the check was here for a reason
+	if(!iscarbon(target))
 		return TRUE
```

Comment:
If it's possible but should never happen, you should stack_trace it so it fails unit tests (if such a unit test does or every may exist) and so coders can investigate and feex.

Reply:
```suggestion
	// this shouldn't even be possible, but I'm sure the check was here for a reason
	if(!iscarbon(target))
		stack_trace("HEY LISTEN! We are performing a species spec_unarmed attack with a non-carbon user. How did you fuck this up?")
		return TRUE
```